src/components/Comments.js

```diff
@@ -28,9 +30,21 @@ export const Comments = ({id}) => {
         setIsLoading(true)
         })
         .catch((err)=>{
-            console.log(err)
         })
     }
+
+    if(removeComment){
```

Comment:
I would add the code in this if block to your deleteComment function. This function should also handle the state of the list of comments and update it accordingly, which should work for multiple consecutive comment deletions